packages/core-bridge/src/runtime.rs

```diff
@@ -145,13 +146,9 @@ pub fn start_bridge_loop(
                     headers,
                     callback,
                 } => {
-                    let runtime_clone = core_runtime.clone();
                     core_runtime.tokio_handle().spawn(async move {
                         match options
-                            .connect_no_namespace(
-                                runtime_clone.metric_meter().as_deref(),
-                                headers.map(|h| Arc::new(RwLock::new(h))),
-                            )
+                            .connect_no_namespace(None, headers.map(|h| Arc::new(RwLock::new(h))))
```

Comment:
Hmm... does this effectively disable metrics for client calls? Is there an alternative API we should be using? (I haven't kept up-to-date on the recent core changes).

Reply:
Yes, all the Rust changes in this PR should be ignored (as mentioned in PR description). They're just a workaround for a pending upgrade of core-bridge to latest `sdk-core`. I think @mjameswh knows most about it.